examples/webpack/env.d.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference types="vite/client" />
+
+declare module '@vue-router/routes' {
```

Comment:
I think we can remove this part as it's already tested in the vite playground. Let's keep this example to the minimal to test out locally 💯